.github/workflows/check.yaml

```diff
@@ -1,5 +1,5 @@
-name: Check modules on registry.coder.com
-
+name: Health
```

Comment:
the name change seems to be required to get a shorter name for the health check badge.